arangod/VocBase/vocbase.cpp

```diff
@@ -1019,7 +1022,18 @@ void TRI_vocbase_t::inventory(
       result.close();
     }
   }
-
+  result.close();
+  
+  result.add("views", VPackValue(VPackValueType::Array, true));
+  for (auto const& dataSource : dataSourceById) {
+    if (dataSource.second->category() != LogicalView::category()) {
+      continue;
+    }
+    LogicalView const* view = static_cast<LogicalView*>(dataSource.second.get());
+    result.openObject();
+    view->toVelocyPack(result, /*details*/false, /*forPersistence*/true);
```

Comment:
don't forget to dump all collections the view depends on (via visitCollections(...))

Reply:
We are dumping all collection definitions right above that.

Reply:
what about the case when the user requested specific collections/views?

Reply:
actually I looked, we only exclude collections like `_statistics*`, `_frontend` and optionally all other system collections.
We should probably prevent creating views on these anways

Reply:
or better yet, prevent exporting views which reference collections that are not exportable (via visitCollections(...))